client/constants.py

```diff
@@ -25,10 +25,33 @@ class UrlTypes(object):
     YOUTUBE = "youtube"
 
     special_urls = {
-        WIKI: "en.wikipedia.org",
+        WIKI: "wikipedia.org",
         YOUTUBE: "youtube.com"
     }
 
     @staticmethod
     def get_special_url(url_type):
         return UrlTypes.special_urls.get(url_type)
+
+
+class JsonFields(object):
+    COUNT = "count"
```

Comment:
Split into:
FieldKeyword
FieldMediaType

and remove Json from all categorization. Only have the name 'json' in the get_json method